presto-product-tests/src/main/java/io/prestosql/tests/hive/BaseHivePartitioningTest.java

```diff
@@ -1,55 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.prestosql.tests.hive;
-
-import io.prestosql.jdbc.PrestoResultSet;
-import io.prestosql.tempto.BeforeTestWithContext;
-import io.prestosql.tempto.ProductTest;
-import io.prestosql.tempto.query.QueryResult;
-import io.prestosql.tests.querystats.QueryStatsClient;
-
-import javax.inject.Inject;
-
-import java.sql.SQLException;
-
-import static io.prestosql.tempto.query.QueryExecutor.query;
-import static java.lang.String.format;
-
-public abstract class BaseHivePartitioningTest
-        extends ProductTest
-{
-    private QueryStatsClient queryStatsClient;
-
-    @Inject
-    @BeforeTestWithContext
-    public void setUp(QueryStatsClient queryStatsClient)
-    {
-        this.queryStatsClient = queryStatsClient;
```

Comment:
I guess it does not solve the problem completely. I would suspect that every test that is using `QueryStatsClient` is vulnerable of being flaky.

Reply:
Luckily this is the only test that it was using this, so please remove `QueryStatsClient` and friends as well.

Reply:
I'm hesitant. It's not `QueryStatsClient`'s fault and i think `QueryStatsClient` might be valuable.
I want to leave https://github.com/prestosql/presto/issues/2382 open.

I will still remove this class as we should test https://github.com/prestosql/presto/issues/2382 with engine "unit" tests, not product tests though.

Reply:
> I will still remove this class as we should test #2382 with engine "unit" tests, not product tests though.

I agree, but do we have such tests now?

> QueryStatsClient might be valuable.

Correct, but if it is not used then then it a risk that it become broken at some point.

How about https://github.com/prestosql/presto/pull/2385, it does not remove coverage and still uses `QueryStatsClient`?